driver/razerkbd_driver.c

```diff
@@ -1641,6 +1654,14 @@ static ssize_t razer_attr_write_matrix_effect_wave(struct device *dev, struct de
         request.transaction_id.id = 0x3F;
         break;
 
+    case USB_DEVICE_ID_RAZER_ORNATA_V3:
+    case USB_DEVICE_ID_RAZER_ORNATA_V3_ALT:
+        // Direction values are flipped compared to other devices
+        direction ^= ((1<<0) | (1<<1));
```

Comment:
This can be fixed by adding wave_dirs in keyboard.py instead:
Example:
![image](https://github.com/openrazer/openrazer/assets/35467946/8e3f40a8-1038-4928-9c73-80e19d17a149)

Reply:
The wave_dirs are already included for the OV3 in keyboards.py.

https://github.com/openrazer/openrazer/blob/a3ced8526c858bcd795fb781a6a5f151092ed5aa/daemon/openrazer_daemon/hardware/keyboards.py#L765-L774

The direction flip operation was used like as some of the accessories do so in the driver code:

https://github.com/openrazer/openrazer/blob/85e81ae3ba08f2af33031e8a08a4f0ecc6adee91/driver/razeraccessory_driver.c#L673-L678

https://github.com/openrazer/openrazer/blob/85e81ae3ba08f2af33031e8a08a4f0ecc6adee91/driver/razeraccessory_driver.c#L1553-L1558

and the Charging Pad has wave dirs also.

Reply:
Ah sorry I missed that, and it was even explained in the comment already.

Thanks for explaining!